joomla-projects/gsoc22_accessibility-plugin-script
# Made text to speech voice adapt to the website language.

### Summar of changes:

Fixes: https://forum.joomla.org/viewtopic.php?f=808&t=991513&p=3651110
1. Made text-to-speech-language and speech-to-text language change to the currently active language of the site. Default is en-US. 
2. If the language is not supported then tts will not be displayed.


### Testing instruction:

1. Change to a language other than en-US. Check if the voice of the text-to-speech adapts to the site language or not.
2. If the tts is not displayed then can confirm from here: https://jsbin.com/wuqumukesu/edit?js,console,output
that language is not supported by your browser/os.
